src/MPI.jl

```diff
@@ -80,6 +79,10 @@ include("misc.jl")
 
 include("deprecated.jl")
 
+if !isdefined(Base, :get_extension)
```

Comment:
Out of curiosity (and such that I can learn from it): Why isn't this guarded by `@static` as well, e.g.
```julia
@static if !isdefined(Base, :get_extension)
```
?

Reply:
`@static` doesn't do much for top-level expressions, they're always evaluated at precompile time anyway.

Reply:
Ah, thanks a lot for the clarification!